contrib/scoverage/test/src/HelloWorldTests.scala

```diff
@@ -190,14 +190,14 @@ trait HelloWorldTests extends utest.TestSuite {
 
 object HelloWorldTests_2_12 extends HelloWorldTests {
   override def threadCount = Some(1)
-  override def testScalaVersion: String = sys.props("MILL_SCALA_2_12_VERSION")
-  override def testScoverageVersion = sys.props("MILL_SCOVERAGE_VERSION")
+  override def testScalaVersion: String = sys.props.getOrElse("MILL_SCALA_2_12_VERSION", ???)
+  override def testScoverageVersion = sys.props.getOrElse("MILL_SCOVERAGE_VERSION", ???)
   override def testScalatestVersion = "3.0.8"
 }
 
 object HelloWorldTests_2_13 extends HelloWorldTests {
   override def threadCount = Some(1)
-  override def testScalaVersion: String = sys.props("MILL_SCALA_2_13_VERSION")
-  override def testScoverageVersion = sys.props("MILL_SCOVERAGE_VERSION")
+  override def testScalaVersion: String = sys.props.getOrElse("TEST_SCALA_2_13_VERSION", ???)
+  override def testScoverageVersion = sys.props.getOrElse("MILL_SCOVERAGE_VERSION", ???)
```

Comment:
Why this change?
```scala
@ Map(1 -> 1)(2)
java.util.NoSuchElementException: key not found: 2
  scala.collection.immutable.Map$Map1.apply(Map.scala:245)
  ammonite.$sess.cmd0$.<clinit>(cmd0.sc:1)


@ Map(1 -> 1).getOrElse(2, ???)
scala.NotImplementedError: an implementation is missing
  scala.Predef$.$qmark$qmark$qmark(Predef.scala:344)
  ammonite.$sess.cmd1$.$anonfun$res1$1(cmd1.sc:1)
  scala.collection.immutable.Map$Map1.getOrElse(Map.scala:250)
  ammonite.$sess.cmd1$.<clinit>(cmd1.sc:1)
```

Reply:
`sys.props` has a `def default(key: String) = null` override.

Reply:
🤯